cdap-data-fabric/src/test/java/io/cdap/cdap/spi/data/sql/SqlStructuredTableTest.java

```diff
@@ -80,7 +92,7 @@ protected StructuredTableAdmin getStructuredTableAdmin() {
   }
 
   @Override
-  protected TransactionRunner getTransactionRunner() {
+  protected TransactionRunner getTransactionRunner() throws TransactionException {
```

Comment:
Why this change?

Reply:
Added a try-catch to handle it coming from `sqlTransactionRunner.run()`.